packages/cli/src/lib/project/Project.ts

```diff
@@ -79,4 +85,32 @@ export abstract class Project {
       });
     });
   }
+
+  /// Validation
+
+  protected validateManifestLanguage(
+    language: string | undefined,
+    validPatterns: string[]
+  ): void {
+    if (!language) {
+      throw Error(intlMsg.lib_project_language_not_found());
+    }
+
+    const languagePatternValid = (test: string) => {
+      let valid = false;
+      validPatterns.map((x) => (valid = valid || test.indexOf(x) > -1));
+      return valid;
```

Comment:
```suggest
      return validPatterns.every((x) => (test.indexOf(x) > -1));
```

Reply:
In this case, I think the one liner would use `validPatterns.some(...)` instead of `.any(...)`

Reply:
Ah my bad, It's `OR` haha

Reply:
Amazing, thanks so much, did not think of using some here :D